src/main/java/net/sf/jabref/external/DroppedFileHandler.java

```diff
@@ -345,7 +345,7 @@ public void stateChanged(ChangeEvent arg0) {
 
         linkInPlace.setText(Localization.lang("Leave file in its current directory."));
         copyRadioButton.setText(Localization.lang("Copy file to file directory."));
-        moveRadioButton.setText(Localization.lang("Move file to file directory."));
+        moveRadioButton.setText(Localization.lang("Move file to file directory") + '.');
```

Comment:
I know the concept, but in this case I removed one of

"Move file to file directory"
"Move file to file directory."
"Move file to file directory?"

Not sure how to balance the number of strings and correctness...